{"name": "The user should be able to view orders repository", "status": "broken", "statusDetails": {"message": "smtplib.SMTPAuthenticationError: (534, b'5.7.9 Please log in with your web browser and then try again. For more\\n5.7.9 information, go to\\n5.7.9  https://support.google.com/mail/?p=WebLoginRequired d2e1a72fcca58-7023260a9e6sm333239b3a.163 - gsmtp')\n", "trace": "  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\behave\\runner.py\", line 545, in run_hook\n    self.hooks[name](context, *args)\n  File \"Features\\environment.py\", line 108, in after_step\n    s.login(fromaddr, \"E$$pl@2022!#21\")\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\smtplib.py\", line 750, in login\n    raise last_exception\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\smtplib.py\", line 739, in login\n    (code, resp) = self.auth(\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\smtplib.py\", line 662, in auth\n    raise SMTPAuthenticationError(code, resp)\n"}, "steps": [{"name": "And Scroll through menu to view planning option", "status": "broken", "statusDetails": {"message": "smtplib.SMTPAuthenticationError: (534, b'5.7.9 Please log in with your web browser and then try again. For more\\n5.7.9 information, go to\\n5.7.9  https://support.google.com/mail/?p=WebLoginRequired d2e1a72fcca58-7023260a9e6sm333239b3a.163 - gsmtp')\n", "trace": "  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\behave\\runner.py\", line 545, in run_hook\n    self.hooks[name](context, *args)\n  File \"Features\\environment.py\", line 108, in after_step\n    s.login(fromaddr, \"E$$pl@2022!#21\")\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\smtplib.py\", line 750, in login\n    raise last_exception\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\smtplib.py\", line 739, in login\n    (code, resp) = self.auth(\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\smtplib.py\", line 662, in auth\n    raise SMTPAuthenticationError(code, resp)\n"}, "attachments": [{"name": "screenshot", "source": "adcaf66e-a5d9-45fd-8e11-a7b3d57c271e-attachment.png", "type": "image/png"}], "start": 1717046576123, "stop": 1717046594353}, {"name": "And Scroll through menu to view orders menu", "status": "skipped", "start": 1717046594354, "stop": 1717046594354}, {"name": "And Click on orders button", "status": "skipped", "start": 1717046594354, "stop": 1717046594354}], "start": 1717046575113, "stop": 1717046594354, "uuid": "8be3f14c-5045-422a-b0e4-f7718dfab15f", "historyId": "8861ba84460d4043a237208c15b1b0a1", "fullName": "Verify the TMS functionality: The user should be able to view orders repository", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "Orders"}, {"name": "feature", "value": "Verify the TMS functionality"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}